class-00/lab-c/starter-code/styles/base.css

```diff
@@ -0,0 +1,43 @@
+@import url('https://fonts.googleapis.com/css?family=Abel|Oswald:700|Yellowtail|Patrick+Hand');
+
+* {
+  font-family: times;
+}
```

Comment:
generally base.css is the smallest; would really only have `*`. sometimes you'll have more, but it's really only used on things that cover the widest range of elements